system/ui/lib/scroll_panel2.py

```diff
@@ -10,7 +10,7 @@
 
 MIN_VELOCITY = 10  # px/s, changes from auto scroll to steady state
 MIN_VELOCITY_FOR_CLICKING = 2 * 60  # px/s, accepts clicks while auto scrolling below this velocity
```

Comment:
adjust this, not is_touch_valid. you want to be able to tap when moving very slowly